webplugin/css/app/mck-sidebox-1.0.css

```diff
@@ -491,7 +491,142 @@
 .mck-box-footer .mck-btn {
 	font-size: 12px;
 }
+.mck-char-warning{
+	width: auto;
+	height: 62px;
+	margin-bottom: -70px;
+	padding-top:15px;
+	border-radius: 4px;
+	opacity:0;  
+  	background-color: #5451e2;
+}
+.warning-slide-up{
```

Comment:
@hardikparashar add prefix mck- , this is added to avoid css conflict with customer's website css in cases when customer decides to our open source code and add it to their project.

Reply:
done